tensorflow/python/kernel_tests/mkl_ops_test.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""
+  Tests to verify if the conversion of python ops to MKL graph ops happen when
+  MKL is enabled. These tests create graphs to verify the rewrite, and
+  therefore will not test eager rewrites. The tests will be skipped if MKL is
+  not enabled.
+"""
+
+from tensorflow.python.framework import constant_op
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import test_util
+from tensorflow.python.ops import nn_ops
+from tensorflow.python.platform import test
+from tensorflow.core.protobuf import config_pb2
```

Comment:
The import order should be sorted alphabetically. Please move `config_pb2` and `rewriter_config_pb2` to the top. (`tensorflow.core` before `tensorflow.python`.)